src/Xamarin.Android.Build.Tasks/Utilities/NativeAssemblyGenerator/NativeAssemblyGenerator.cs

```diff
@@ -207,7 +207,7 @@ void EnsureTypeMapping ()
 
 		public virtual void WriteFileTop ()
 		{
-			WriteDirective (".file", QuoteString (fileName));
+			WriteDirective (".file", QuoteString (fileName.Replace ("\\", "\\\\")));
```

Comment:
Should this instead just replace `\` with `/`?

Reply:
I'd rather use native separators